internal/pkg/describe/service.go

```diff
@@ -296,6 +299,17 @@ func (d *ecsServiceDescriber) ServiceConnectDNSNames() ([]string, error) {
 	return service.ServiceConnectAliases(), nil
 }
 
+func (d *ecsServiceDescriber) RollbackAlarmNames() ([]string, error) {
```

Comment:
doc-comments 🥺

Reply:
Ha! 🤦🏼 I deleted them when I started making it private. Will reinstate! Thanks for catching.